source/manual/ruby.html.md

```diff
@@ -32,17 +32,33 @@ We set up rbenv differently depending on what's going on:
 
 ## Add a new Ruby version in puppet
 
-Build a package using [fpm](debian-packaging.html#fpm) and make it
-available using our aptly repository.
+You will need to build a new [fpm](debian-packaging.html#fpm) package with the new Ruby version.
+This package can then be copied to Aptly machine, and the new version added to puppet.
 
+### Building the fpm package.
+
+- Add a new recipe for the ruby version in [Packager](packager).
+The folder name will be the Ruby version, and contain a `recipe.rb` file. See previous entries for examples.
+The recipe will require the SHA256 of the version's `tar.gz` file.
+- Once the Packager change is merged, [build the package](jenkins).
+The Jenkins job will produce a `.deb` package in the `Last Successful Artifacts`.
 The package name includes the version of Ruby (eg `rbenv-ruby-2.3.1`) because
 we need to install multiple versions of Ruby at the same time.
 
-Once it's available as a package you can
+- To test the package, download the file and run `sudo dpkg -i path/to/the/downloaded/package.deb` in the VM.
+Run `ruby -v` to see if your new Ruby version is available
```

Comment:
`ruby -v` would only return the current version of ruby, and that wouldn't be the one just installed. In order to check that everything's ok, we probably want to:
 - list the available rubies (`rbenv versions`) and see that the newly installed one is there
 - set the new one to be used (`rbenv local 2.6.1`) and see that the switch was successful (`ruby -v`)

Reply:
oh yeah.  whoops!